ampl-converter/src/main/java/com/powsybl/ampl/converter/AmplNetworkWriter.java

```diff
@@ -490,7 +490,7 @@ private void writeDanglingLineMiddleBuses(AmplExportContext context, TableFormat
                 context.busIdsToExport.add(middleBusId);
                 int middleBusNum = mapper.getInt(AmplSubset.BUS, middleBusId);
                 int middleVlNum = mapper.getInt(AmplSubset.VOLTAGE_LEVEL, middleVlId);
-                SV sv = new SV(t.getP(), t.getQ(), b != null ? b.getV() : Double.NaN, b != null ? b.getAngle() : Double.NaN).otherSide(dl);
+                SV sv = new SV(t.getP(), t.getQ(), b != null ? b.getV() : Double.NaN, b != null ? b.getAngle() : Double.NaN, Branch.Side.ONE).otherSide(dl, true);
```

Comment:
Not sure I understand why you need to specify the side for dangling lines...

Reply:
It is not necessary if we specify the splitShuntAdmittance to true, but to avoid a future misuse we prefer to specify always the side. We think that the side must be always specified.

Reply:
Although dangling lines in IIDM only have one terminal (one _side_), they are modelling branches with two ends. So even for dangling lines, the SV utility class needs to know for which end are you giving the voltage and flow, when you want to calculate the _otherSide_ magnitudes. For dangling lines the _network_ end is always considered side `ONE` and the _boundary_ end is side `TWO`.